Document tag_value_ids parameter for resource creation endpoints (ABAC) (#4510)
langchain-ai/docs: src/langsmith/abac.mdx

}
```

## Tag resources at creation time

> When ABAC policies are active, resources are access-controlled based on their tags. To make sure a resource is protected from the moment it is created — without any window where it is untagged — you can supply tags directly in the creation request using the `tag_value_ids` parameter.

Reviewer: ```suggestion
When ABAC policies are active, resources are access-controlled based on their tags. To make sure a resource is protected as soon as it is created, you can supply tags directly in the creation request using the `tag_value_ids` parameter.
```
